index.js

```diff
@@ -48,6 +37,7 @@ function getListLocales() {
         reject(e);
       });
     } else {
+      supportedLocales = supportedLocales.split(",").map(item => item.trim());
```

Comment:
I'd say this is a reasonable amount of defensive programming to include and be worth a quick loop.